src/components/[guild]/EditGuild/components/UrlName.tsx

```diff
@@ -30,9 +30,14 @@ const UrlName = () => {
         <Input
           {...register("urlName")}
           onChange={(event) => {
-            setValue("urlName", slugify(event.target.value), {
-              shouldDirty: true,
-            })
+            setValue(
```

Comment:
What if we'd just define a custom validator inside the `register` function which checks if `value === slugify(value)` and we display an "Invalid URL name" error message otherwise? 👀 
That way we could get rid of the custom `onChange` handler and we wouldn't modify the user input on-the-fly. This way the user would be able to write a dash, but they wouldn't be able to submit the form, since it would be invalid that way. Wdyt?

Reply:
Sure thing! Using validation is much more gentle on the user than overwriting the input value. 😉